tests/integration/example_test_env_vars.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 # Cachito API endpoint
-api_url: https://cachito.example.com/api/v1
+api_url: https://cachito.${CACHITO_ENV_SUBURL}.com/api/v1
```

Comment:
How about just making the whole thing an environment variable that is called `CACHITO_API_URL`?

Reply:
Sure, I could do it. That might be easier. I just didn't want to change too much and went with this but I like your suggestion more.

Reply:
@mprahl  I made necessary changes in the respective MR as well. Please re-review. Resolving this.